src/types/ThemeBreakpoints.ts

```diff
@@ -0,0 +1,7 @@
+import { defaultBreakpoints } from '../defaultBreakpoints';
+
+type Transform<T> = {
+  [P in keyof T]: string;
+};
+
+export interface ThemeBreakpoints extends Transform<typeof defaultBreakpoints> {}
```

Comment:
What's the purpose of these `Transform` generics?

Reply:
Create a type based on the keys of defaultBreakpoints

Reply:
Curious why we need a separate type in addition to `defaultBreakpoints`?

Reply:
I dont think we're able to dynamically create a theme (createTheme.ts) if we use types that are constants.